rest/rest-sqs/src/main/scala/org/elasticmq/rest/sqs/CreateQueueDirectives.scala

```diff
@@ -13,6 +13,7 @@ import CreateQueueDirectives._
 
 trait CreateQueueDirectives { this: ElasticMQDirectives with QueueURLModule with AttributesModule with SQSLimitsModule =>
 
+  // DeadLettersQueue currently not supported via http call due to current API limitations. Consider moving to JSON?
```

Comment:
Aren't DLQ's configured via attributes passed during queue creation? The docs (http://docs.aws.amazon.com/AWSSimpleQueueService/latest/APIReference/API_CreateQueue.html) mention `RedrivePolicy`.  Where's the JSON?